### PR TITLE
upgrading the configuration variables

### PR DESCRIPTION
**Pull request by 1sarthakbhardwaj**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -430,7 +430,9 @@ set(Boost_ADDITIONAL_VERSIONS
   "1.61.1" "1.61.0" "1.61"
   "1.60.1" "1.60.0" "1.60"
   "1.59.1" "1.59.0" "1.59"
-  "1.58.1" "1.58.0" "1.58")
+  "1.58.1" "1.58.0" "1.58"
+  "1.74.0" "1.73"  
+  "17.4.0" "17.4"	)
```
Comment (zoq) [inline CMakeLists.txt:435]: Do you mind to add the lines to the top, we are trying to keep the versions in order.

**Review by zoq — COMMENTED** [review]
Thanks for the fast patch, very much appreciated.

**Comment by 1sarthakbhardwaj** [comment]
Alright you can as your wish

**Comment by zoq** [comment]
Closing favour of https://github.com/mlpack/mlpack/pull/2605

**Pull request closed without merging**